src/icon-button.test.interactions.ts

```diff
@@ -0,0 +1,16 @@
+import { expect, fixture, html } from '@open-wc/testing';
+import GlideCoreIconButton from './icon-button.js';
+
+it('is accessible', async () => {
+  const host = await fixture<GlideCoreIconButton>(
+    html`<glide-core-icon-button label="Label">
+      <div>Icon</div>
+    </glide-core-icon-button>`,
+  );
+
+  host.ariaDescription = 'Description';
```

Comment:
Gives us test coverage for [this](https://github.com/CrowdStrike/glide-core/pull/871/files#diff-872489a441290f93abf84a5ec9dfe19fa9e24d73f4461aef791316eb77af3a2dR65-R69). Also not a bad thing to test. Otherwise, I'd have made it a "has [thing] coverage" test.